common/deliverclient/util.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
don't we have any util package in our entire codebase we can move this function to?

Reply:
It got promoted from `orderer/common/cluster/util.go` to `common/deliverclient/util.go`, where I plan to put all the stuff that is common to both orderer and peer in the context of the block puller. Can't put it in package `common/util` nor `protoutil` because of import cycles. If you have other ideas I'd love to hear.

Reply:
I suppose the import cycle is because of `github.com/hyperledger/fabric/common/configtx`? Then can't you move it there?

Reply:
However if you think there are additional things to be placed here, I guess we can leave it as is.

Reply:
In terms of import cycles it can be placed in `common/configtx`, but I think it is not a good place to put code that parses blocks, as that package deals only with individual txs. I prefer to leave it where it is for the time being. I think that during the refactoring that is about to happen when we overhaul the orderer, several additional things will get moved into that util.go file.